src/Console/TinkerCommand.php

```diff
@@ -53,6 +54,11 @@ public function fire()
         $shell->addCommands($this->getCommands());
         $shell->setIncludes($this->argument('include'));
 
+        (new ClassAliasAutoloader(
+            $shell, 
+            $this->getLaravel()->basePath().'/vendor/composer/autoload_classmap.php'
```

Comment:
I think it looks ok. I think laravel 4 had a vendor folder constant, but I think we got rid of it, so this is fine, so long as `basePath` exists on both laravel and lumen application classes. :)